packages/application-generic/src/profiling/profiling.controller.ts

```diff
@@ -0,0 +1,33 @@
+import { Controller, Get, HttpException } from '@nestjs/common';
+import { ApiExcludeController } from '@nestjs/swagger';
+import { PinoLogger } from '../logging';
+import { ProfilingService } from './profiling.service';
+
+@Controller('/debug/pprof')
+@ApiExcludeController()
```

Comment:
I guess these won't be accessible from the outside right?

Reply:
Currently, I have not hooked them to the api yet as with AWS ALB it is not able to block these request but istio can. 
I can also have this set up where it is conditionally enabled.